webapp/components/wpt-amend-metadata.js

```diff
@@ -335,7 +344,11 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
     }
 
     for (const key in browserMap) {
-      this.displayedMetadata.push({ product: key, url: '', tests: browserMap[key] });
+      let node = { product: key, url: '', tests: browserMap[key] };
+      if (key === '') {
+        node['label'] = '';
```

Comment:
Can you add a comment about this case? Why do we need to set the label to the empty string if there is no product?

Reply:
Added. Basically we will set a `label` field when it is a test-level triage (product is empty)